dolfyn/io/rdi.py

```diff
@@ -41,15 +41,15 @@ def read_rdi(fname, userdata=None, nens=None, keep_orient_raw=False):
     adp_data : :class:`ADPdata <dolfyn.adp.base.ADPdata>`
 
     """
-    # userdata is not used here.
+    # userdata is not used here. CLARIFY COMMENT..
```

Comment:
That's an old comment. Delete this line.

Reply:
Done.